apollo-gradle-plugin-incubating/src/test/kotlin/com/apollographql/apollo/gradle/test/ConfigurationTests.kt

```diff
@@ -351,6 +351,59 @@ class ConfigurationTests {
     }
   }
 
+  @Test
+  fun `sources can be overridden in compilationUnits`() {
+    withSimpleProject("""
+      apollo {
+        schemaPath = "com/some/other/schema.json"
```

Comment:
```suggestion
        schemaFile = file("com/some/other/schema.json")
```

Reply:
Changing to this actually makes the test go green. I am not even sure if we have a bug to fix. 🤷‍♂️